terraform/09-video-hearing-core/modules/KeyVaults/main.tf

```diff
@@ -1,5 +1,5 @@
 locals {
-  environment = var.environment
+  environment   = var.environment
   current_year  = formatdate("YYYY", timeadd(timestamp(), "8760h"))
```

Comment:
Can we remove these locals as we are hardcoding the expiry date now?

Reply:
done